numpy/_core/strings.py

```diff
@@ -1373,6 +1373,14 @@ def _splitlines(a, keepends=None):
     --------
     str.splitlines
 
+    Examples
```

Comment:
Unless there's some magic that uses the docstring of the private function in the public function, not sure if this is needed here?